packages/dev/core/src/Misc/precisionDate.ts

```diff
@@ -8,10 +8,11 @@ export class PrecisionDate {
      * Gets either window.performance.now() if supported or Date.now() else
      */
     public static get Now(): number {
-        if (IsWindowObjectExist() && window.performance && window.performance.now) {
-            return window.performance.now();
-        }
-
-        return Date.now();
+        return PrecisionDate._Now();
     }
+
+    /**
+     * Since Now is a property, we're using a private variable with a lambda to check which now function to use once
+     */
+    private static readonly _Now = IsWindowObjectExist() && window.performance && window.performance.now ? () => window.performance.now() : () => Date.now();

```

Comment:
This should also just be a local variable. No need to store on the class.

Reply:
I believe that we talked about this offline and we need to set up this way due to how `Now` is used.

Reply:
Moved out to const variable